distributed/cli/tests/test_dask_submit.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import print_function, division, absolute_import
+from distributed.utils_test import valid_python_script, popen, invalid_python_script
+
+def test_dask_submit_cli(valid_python_script, invalid_python_script):
+    with popen(['dask-scheduler', '--port', '8989']) as sched:
+        with popen(['dask-submit', '127.0.0.1:8788', str(valid_python_script)]) as submit:
+            for x in range(5):
+                line = submit.stdout.readline()
+                if b'hello world!\\n' in line:
+                    break
+
+        with popen(['dask-submit', '127.0.0.1:8788', str(invalid_python_script)]) as submit:
+            for x in range(5):
+                line = submit.stdout.readline()
+                if b'Traceback' in line:
+                    break
+
+
+def test_submit_a_distributed_script(executor_contract_script):
```

Comment:
Does it fail when run in isolation?  Is it possible that this port is occupied by a previous test?  Does sched have anything interesting in its stderr logs?